infra/app/service/main.tf

```diff
@@ -90,3 +90,17 @@ module "service" {
     }
   } : null
 }
+
+module "monitoring" {
+  source = "../../modules/monitoring"
+  #Module take ecs cluster name to link all alerts with ECS monitored cluster
+  ecs_name = module.service.cluster_name
+
+  #To enable alert uncomment appropriate line
+  cpu_threshold = "80"
+  #memory_util_threshold          = "70"
+  #task_health_threshold          = "100"
+  #task_placement_threshold       = "0"
+  #service_availability_threshold = "1"
+  #network_connectivity_threshold = "1"
```

Comment:
I wrote you an answer in slack and will duplicate it here:
Hey 
[@loren](https://nava.slack.com/team/U03MJABB4)
 as for one of your questions in pr regarding survey. I did ask about ecs monitoring across different projects and received pretty limited feedback in regards to metric monitoring. Some of the teams rely on cpu alerts and they even have autoscaling rules based on that. but most of the teams just do only application monitoring with NewRelic. So I did research around, read few articles about best practices for ecs monitoring and come up with the list of alerts. I do no think projects didn’t use alerts in cloudwatch, because they are not useful, but probably because newRelic was prioritized in CMS. At least it was like that with ITOPS.
so just to summarize, most common alerts were cpu and memory usage.